### PR TITLE
Caladero and Spain work

### PR DESCRIPTION
**Pull request by Xerodoeht**
Also Bosnia, Morocco, and some other things.

### CODE REVIEW

**Comment by Glloyd** [comment]
There's a lot here. In the future maybe try to split different country (or even different continent) content across different PRs or branches for ease of review.

**Comment by Xerodoeht** [comment]
That is a good point, I apologize

File: events/dvg_caladero.txt [hunk 1]
```diff
@@ -127,3 +130,679 @@ dvg_caladero.3 = {
...
+
+	duration = 3
+
+	trigger = {
+	}
+	
+	#As Is
+	option = {
+		name = dvg_caladero.10.a
+		default_option = yes
```
Comment (MadJamez) [inline events/dvg_caladero.txt:482]: You have 2 default options in this event.

File: events/dvg_caladero.txt [hunk 2]
```diff
@@ -127,3 +130,679 @@ dvg_caladero.3 = {
...
+
+	duration = 3
+
+	trigger = {
+	}
+	
+	#As Is
+	option = {
+		name = dvg_caladero.11.a
+		default_option = yes
```
Comment (MadJamez) [inline events/dvg_caladero.txt:552]: 2 default options. Should only have 1 default.

**Comment by MadJamez** [comment]
@Xerodoeht I played around with it a bit. And these changes look cool.
Only 1 remark. When you become a republic as Caladero. Maybe we can switch ideology of landowners to ideology_republican_paternalistic? Now they support the monarchy. But when switching ideology like this, they are supporting a republic instead. Or maybe some other ideology to switch from paternalistic to make them more loyal to the republic?

It is something we should maybe do for countries starting as republics, or become one by event? I already changed the ideology of Venice and Novgorod in my russia branch.

**Review by MadJamez — CHANGES_REQUESTED** [review]

**Review by MadJamez — CHANGES_REQUESTED** [review]
Some errors in log:
[17:58:35][pdx_persistent_reader.cpp:252]: Error: "Unknown trigger type: coa_def_scandinavian_ensign_trigger, near line: 9
Unknown trigger type: coa_def_scandinavian_ensign_trigger, near line: 10" in file: "common/scripted_triggers; coa_def_colonial_ensign_trigger" near line: 12

Variable 'dvg_martin_is_glad' is set but is never used.

**Review by MadJamez — APPROVED** [review]

**Pull request merged**